de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/user/UserRepository.java

```diff
@@ -75,4 +75,16 @@ public User save(@NonNull final User user)
 				.build();
 	}
 
+	public User updateEmail(final User user, final String newMail)
```

Comment:
i think the repository's job should be to load and store users, and it should not contain BL like this. @teosarca wdyt?

Reply:
agree

Reply:
done